src/GameStore.js

```diff
@@ -36,32 +36,40 @@ export class GameStore {
         return this.heroes
     }
     toggleShowItems = () => {
-        this.showItems = !this.showItems;
-        this.showSkills = false
-        this.showActions = false
-        this.activeAction = null
-        this.activeSkill = null
-        this.activeItem = null
+        runInAction(() => {
+            this.showItems = !this.showItems;
+            this.showSkills = false
+            this.showActions = false
+            this.activeAction = null
+            this.activeSkill = null
+            this.activeItem = null
+        })
     }
     toggleShowActions = () => {
-        this.showActions = !this.showActions;
-        this.showSkills = false
-        this.showItems = false
-        this.activeAction = null
-        this.activeSkill = null
-        this.activeItem = null
+        runInAction(() => {
+            this.showActions = !this.showActions;
```

Comment:
looks like you're doing the same function as above, why not make a "resetCharacterStatus" function or something like that